### PR TITLE
Rename Pivot to Tabs, Part 1: Rename files and update imports

### PR DESCRIPTION
**Pull request by behowell**
#### Pull request checklist

- [ ] Addresses an existing issue: Fixes #0000
- [x] Include a change request file using `$ yarn change`

#### Description of changes

This is the first of two PRs for renaming the Pivot component to Tabs. This PR only renames files and updates the import/export statements to refer to the new file. There will be a follow-up PR that actually renames the components and properties.

Splitting it out into two PRs allows git to track the renames so that source history is preserved.

Details about the API changes from this rename are here: https://hackmd.io/jDytiF_kQuW2hNELbxgNqA

### CODE REVIEW

File: packages/react/src/Pivot.ts [hunk 1]
```diff
@@ -1 +0,0 @@
-export * from '@fluentui/react-tabs/lib/Pivot';
```
Comment (xugao) [inline packages/react/src/Pivot.ts:1]: we should probably add `Tab` but keeping `Pivot` as is and deprecate. let's follow up tmr

Comment (behowell) [reply packages/react/src/Pivot.ts:1]: I'm planning to export the `Pivot` component from Tabs.ts as a deprecated alias of `Tabs`, in Part 2. As of this PR, the component itself is still named `Pivot`.

If there's a reason to keep the _file_ Pivot.ts around, I can add it back. I suppose it would help avoid problems with places that import from the Pivot.ts file directly.

Comment (ecraig12345) [reply packages/react/src/Pivot.ts:1]: We should keep `packages/react/src/Pivot.ts` since we consider that part of the public API that people are allowed to import directly from (maybe also `packages/react-tabs/src/Pivot.ts`).

File: apps/public-docsite-resources/src/components/pages/PivotPage.tsx [hunk 2]
```diff
@@ -1,7 +1,7 @@
 import * as React from 'react';
```
Comment (khmakoto) [inline apps/public-docsite-resources/src/components/pages/PivotPage.tsx:1]: Since we're updating file names here should we also rename this to `TabsPage`? Similar comment for other files below.

File: packages/react-tabs/src/components/Pivot/index.ts [hunk 3]
```diff
@@ -1,5 +1 @@
-export * from './Pivot';
-export * from './Pivot.base';
-export { PivotItem } from './PivotItem';
-export * from './Pivot.types';
-export * from './PivotItem.types';
+export * from '../Tabs/index';
```
Comment (khmakoto) [inline packages/react-tabs/src/components/Pivot/index.ts:1]: Is this file still needed? Couldn't you just have the `Pivot.ts` file under `src` refer to `Tabs/index` instead of keeping this file around? That way we could remove the `Pivot` folder completely.

Comment (behowell) [reply packages/react-tabs/src/components/Pivot/index.ts:1]: I was planning to use this file in Part 2 to re-export the Pivot, etc. names (as aliases of Tabs, etc.). Although I suppose I can just use react-tabs/src/Pivot.ts to do that...

**Review by khmakoto — COMMENTED** [review]
You probably also need to update references to PivotPage in `AppDefinition.tsx`.

File: packages/react/src/Tabs.ts [hunk 4]
```diff
@@ -0,0 +1 @@
+export * from '@fluentui/react-tabs/lib/Tabs';
```
Comment (xugao) [inline packages/react/src/Tabs.ts:1]: let's not export `Tabs` yet until the all the renaming are done and we are ready to ship with it without futher breaking changes. Also, how are you planning to rename the props? `Pivot` needs to stay as is.

File: packages/react-tabs/src/components/Tabs/Tabs.test.tsx [hunk 5]
```diff
@@ -21,10 +21,11 @@ describe('Pivot', () => {
...
 
-  isConformant({
-    Component: Pivot,
-    displayName: 'Pivot',
-  });
+  // TODO re-enable this test after renaming Pivot to Tabs
+  // isConformant({
+  //   Component: Pivot,
+  //   displayName: 'Pivot',
+  // });
```
Comment (xugao) [inline packages/react-tabs/src/components/Tabs/Tabs.test.tsx:28]: we should not disable all the conformance tests. can cherry-pick the ones which make sense to disable

**Comment by ecraig12345** [comment]
Somewhere (can't find it now) I saw a question about naming of PivotItem/TabPanel, and the new name choice being mainly based on aria. Have you checked what other frameworks call it?

**Comment by behowell** [comment]
Abandoning this PR in favor of #15782, which creates Tabs as a copy of Pivot, rather than renaming it. This'll make it easier to make future API updates to Tabs without breaking existing users of Pivot.
